nucypher/cli/literature.py

```diff
@@ -55,7 +55,7 @@
 * Ursula Node Operator Notice *
 -------------------------------
 
-By agreeing to stake {str(value)} ({str(value.to_nunits())} NuNits):
+By agreeing to stake {tokens} ({nunits} NuNits):
```

Comment:
I think I may have preferred the previous use of `value`. I like having the one value passed, and the conversion to NuNits being automatically performed. I'm interested in your reasoning for the change...?